### DISCUSSION TITLE
The unnecessary blank

### DISCUSSION THREAD

**Discussion by dingdongliang** [quasarframework/quasar#8988] [General - Components / Directives / etc] [answered]
https://codepen.io/dyenigma/pen/VwPVgaQ?editors=1010

![image](https://user-images.githubusercontent.com/11569220/115701878-d7751e00-a39a-11eb-8deb-ac9278556cb6.png)

How can I solve it? thanks a lot

**Comment by rstoenescu** [comment, marked as answer]
Remove the `style="width:300px"` on the QItemLabel and add `style: 'width:300px'` on the column definition. Might even be better to play with max-width instead? https://codepen.io/rstoenescu/pen/dyNQEBQ?editors=1010

Your problem lies in the fact that you're forcing a fixed width on the content. So the ellipses are working correctly, but it's not what you want.

**Reply by dingdongliang** [reply]
Perfect solution，The 'max-width' is  what I want。
Thank you very much! You are so warmhearted people!
